alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/monitors/package-info.java

```diff
@@ -9,4 +9,3 @@
  * This package contains generic monitors for various Environments.
  */
 package it.unibo.alchemist.boundary.monitors;
```

Comment:
Uhmm... checkstyle should complain about the missing newline

Reply:
This is odd, I see a newline in IDEA but not in Gedit. I ended up deleting the "newline" in IDEA and adding it in Gedit and it git recognizes it as a change. I guess it was some weird character instead of a proper LF, but somehow it eluded checkstyle.

Reply:
weird